I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 392, in script
    pc "I think everything is in order."
Exception: Sayer b'pc' is not a function or string.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 392, in script
    pc "I think everything is in order."
  File "C:\Users\jishe\Desktop\New folder (3)\renpy-8.3.7-sdk\renpy\ast.py", line 2586, in execute
    Say.execute(self)
  File "C:\Users\jishe\Desktop\New folder (3)\renpy-8.3.7-sdk\renpy\ast.py", line 600, in execute
    raise Exception("Sayer %s is not a function or string." % self.who.encode("utf-8"))
Exception: Sayer b'pc' is not a function or string.

Windows-10-10.0.26100 AMD64
Ren'Py 8.3.7.25031702
Drilonis 1.0
Mon Jul 14 18:45:21 2025
